### PR TITLE
fix: update text component styling

### PR DESCRIPTION
**Pull request by 2020Deeya**
# Description

Update text styles to make sure consumers can override the styles in the text component.

## Links

https://jira-eng-gpk2.cisco.com/jira/browse/MOMENTUM-416

### CODE REVIEW

File: packages/components/src/components/text/fonts.styles.ts [hunk 1]
```diff
@@ -1,231 +1,244 @@
...
+  :host([tagname='h2']),
+  :host([tagname='h3']),
+  :host([tagname='h4']),
+  :host([tagname='h5']),
+  :host([tagname='h6']),
+  :host([tagname='p']),
+  :host([tagname='small']),
+  :host([tagname='span']),
+  :host([tagname='div']) {
+    all: unset; /* This removes default styles applied by the browser */
```
Comment (cscheffauer) [inline packages/components/src/components/text/fonts.styles.ts:14]: issue: we should not override the browser defaults here if possible, if the browser has a default we should just keep it.

**Review by cscheffauer — CHANGES_REQUESTED** [review]
general issue: the expectation is that this PR doesn't change the screenshots at all

File: packages/components/src/components/text/fonts.styles.ts (hunk 1 above)
Comment (2020Deeya) [reply packages/components/src/components/text/fonts.styles.ts:14]: yes, Screenshots changes are temporary. They will be reverted back. It was to pushed to check something.

**Comment by 2020Deeya** [comment]
Fresh PR is created instead of this. Hence, closing this one.

**Pull request closed without merging**